server/randomNameGenerator.go

```diff
@@ -109,15 +111,23 @@ var ADJECTIVE = []string{"Abominable", "Accurate", "Adorable", "All", "Alleged",
 
 var LETTERS = []rune("abcdefghijklmnopqrstuvwxyz")
 
+func randomInt(max int) int {
+	value, err := rand.Int(rand.Reader, big.NewInt(int64(max)))
+	if err != nil {
+		mrand.Seed(time.Now().UnixNano())
+		return mrand.Intn(max)
```

Comment:
I added this fallback here to ensure success in the random number generation, but should never be executed in an environment where the crypto random generation is available.

Reply:
`crypto/rand` should always be available so I think this can be safely simplified

Reply:
That sounds like true, but `crypto/rand` `Int` function, returns and `big.Int` or an `error` so, there are cases where the function can fail, more concretely when it is trying to read the `rand.Reader` and it returns an error. So.. I guess is a really weird case, but for me sounds more secure and reliable to leave this "fallback code".

Reply:
We should never use an insecure fallback. The right thing to do is to fail hard so if there's an environment where this isn't working, we notice right away. I have war stories about this exact pattern going terribly wrong and causing major incidents.

The point is: if there's some odd corner case, we absolutely need to know when it occurs and not hide it.

Reply:
Good point, then we should panic here. In theory, if that happens, we will end up with a jitsi plugin not responding from the server side, so is going to be obious enough, but is not going to affect the whole mattermost service :)

Reply:
Done